morph/react-native/block-wrap.js

```diff
@@ -22,6 +22,7 @@ export const enter = (node, parent, state) => {
       `<${block}
           activeOpacity={0.7}
           onPress=${wrap(node.action)}
+          ${node.disabled ? `disabled=${wrap(node.disabled)}` : ''}
```

Comment:
From [this comment below](https://github.com/viewsdx/morph/pull/16/files#r151098778), perhaps we would want to get the `isDisabled` prop here instead?